apps/src/sites/studio/pages/levels/levelGroup.js

```diff
@@ -53,10 +51,9 @@ window.initLevelGroup = function (levelCount, currentPage, lastAttempt) {
         app: subLevel.getAppName(),
         allowMultipleSends: true,
         level: subLevelId,
-        result: subLevelResult,
-        pass: subLevelResult,
-        testResult: testResult,
-        submitted: submitted,
+        result: true,
+        testResult: TestResults.UNVALIDATED_SUBLEVEL,
+        submitted: subLevelResult.submitted || false,
         onComplete: handleSublevelComplete
```

Comment:
This is probably the most dramatic change.. we're confident that it doesn't regress any previous behaviour?